### PR TITLE
Use new ssl certificate kickstart options

### PR DESCRIPTION
**Pull request by larskarlitski**
The `repo` and `url` commands can now contain --sslcacert,
--sslclientcert, and --sslclientkey options. Use those when accessing
repositories.

### CODE REVIEW

**Comment by jkonecny12** [comment]
Hi @larskarlitski, I see a few problems. First you have to link a bug by adding ``Resolves: rhbz#<bugnumber>`` if you want to get this to RHEL-8. Second are you sure this pykickstart version will be available there?

**Comment by M4rtinK** [comment]
Yep, first pykickstart 3.19-1 needs to be built, pass the gating and be added to an errata. Only then we can do our build.

**Comment by larskarlitski** [comment]
Thanks @jkonecny12, I've added the bug number. Indeed, we need to wait for pykickstart to land. That change is here: https://github.com/dcantrell/pykickstart/pull/263

**Comment by larskarlitski** [comment]
Update: pykickstart version and import the RHEL8 versions of classes.

File: anaconda.spec.in [hunk 1]
```diff
@@ -37,7 +37,7 @@ Source0: %{name}-%{version}.tar.bz2
 %define libxklavierver 5.4
 %define mehver 0.23-1
 %define nmver 1.0
-%define pykickstartver 3.16.2-1
+%define pykickstartver 3.16.3-2.1
```
Comment (jkonecny12) [inline anaconda.spec.in]: Please fix the pykickstart version.
First release `2.1` is probably not correct and second there will be a new pykickstart build so the version will probably change.

Comment (jkonecny12) [reply anaconda.spec.in]: The new version should be `3.16.4-1`

**Comment by rvykydal** [comment]
Ported to master with https://github.com/rhinstaller/anaconda/pull/1745

**Pull request merged**